### PR TITLE
[AC-1680] Add manage property to collection view and response models

### PR DESCRIPTION
**Pull request by eliykat**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

The manage property was added to the `CollectionAccessSelectionView`, but it was never added to the primary collection view even though it is being sent down from the server.

We need to add the manage property to both `CollectionDetailsResponse` and `CollectionView` models in the Clients (used in sync data) so it can be used for future client side work.

There are no server changes as it the flag is already being [sent down in the sync data](https://github.com/bitwarden/server/blob/4e7b9d2edd0d1a70df5cc6a1b634f4f7a30af4e9/src/Api/Vault/Models/Response/SyncResponseModel.cs#L38-L39).

## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: libs/common/src/vault/models/domain/collection.ts [hunk 1]
```diff
@@ -27,8 +28,9 @@ export class Collection extends Domain {
         externalId: null,
         readOnly: null,
         hidePasswords: null,
```
Comment (eliykat) [inline libs/common/src/vault/models/domain/collection.ts:30]: I don't think `hidePasswords` should be here, as it 's not actually sent in the response model, so in theory at least it won't ever be set properly. However it's out of scope to mess with here and I don't want to increase the scope of regression testing.

**Review by shane-melton — APPROVED** [review]

File: libs/common/src/vault/models/domain/collection.ts (hunk 1 above)
Comment (djsmith85) [reply libs/common/src/vault/models/domain/collection.ts:30]: @eliykat It's actually missing from the ResponseModel: https://github.com/bitwarden/server/pull/3125

**Comment by vincentsalucci** [comment]
@eliykat - Looks like you'll need to update the `collection.spec.ts` tests 👍

**Comment by eliykat** [comment]
Oops! My brain filtered that out as the usual Enforce PR Labels workflow failing.

**Review by shane-melton — APPROVED** [review]

**Pull request merged**